internal/ls/findallreferences.go

```diff
@@ -1040,9 +1020,9 @@ type refState struct {
 	result        []*SymbolAndEntries
 
 	inheritsFromCache            map[inheritKey]bool
-	seenContainingTypeReferences seenTracker[*ast.Node] // node seen tracker
-	// seenReExportRHS              seenTracker[*ast.Node] // node seen tracker
-	// importTracker                ImportTracker
+	seenContainingTypeReferences *core.Set[*ast.Node] // node seen tracker
```

Comment:
You can just make these `core.Set`, they'll auto initialize (and then you can drop it from the struct literal)